src/plugins/bullmq.ts

```diff
@@ -0,0 +1,40 @@
+import { FastifyInstance } from 'fastify';
+import fp from 'fastify-plugin';
+
+import { BullMQAdapter } from '@bull-board/api/bullMQAdapter';
+import { createBullBoard } from '@bull-board/api';
+import { FastifyAdapter } from '@bull-board/fastify';
+
+import { MOVE_CONTACT_QUEUE, queueManager } from '../shared/queues';
```

Comment:
i don't love that `MOVE_CONTACT_QUEUE` needs to be imported here. could it just be the default for a call to `getQueue()`?